float32 speed_l
float32 speed_r
float32 odometry_l
float32 odometry_r
# float32 gyro_x
# float32 gyro_y
# float32 gyro_z
# float32 accel_x
# float32 accel_y
# float32 accel_z
float32 vin
float32 roll
float32 pitch
float32 yaw
float32 angular_velocity